android/java-authenticator/README.md

```diff
@@ -0,0 +1,26 @@
+<p align="center">
+  <a href="https://github.com/ForgeRock/sdk-sample-apps">
+    <img src="https://cdn.forgerock.com/logo/interim/Logo-PingIdentity-ForgeRock-Hor-FullColor.svg" alt="Logo">
+  </a>
+  <hr/>
+</p>
+
+## Samples
+
+Ping provides these Android samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
+
+### Integrate with PingOne Advanced Identity Cloud / PingAM:
+
+To try out the Ping Android SDK please look at one of our samples:
+
+WIP
+
+
+### Integrate with PingOne DaVinci:
+
+- Samples coming soon
+
+## Requirements
+
+iOS
+- Latest version of Android Studio
```

Comment:
Add to requirements:
Ping Advanced Identity Cloud or Ping AM 7.1 and above
Android API level 23+
Google Firebase project (google-services.json)